Deploy-CAM.ps1

```diff
@@ -2202,7 +2268,86 @@ else {
+    }
+
+    $vnetConfig = @{}
+    $vnetConfig.vnetName = $vnetName
+    $vnetConfig.CSsubnetName = $CloudServicesSubnetName
+    $vnetConfig.GWsubnetName = $GatewaySubnetName
+    $vnetConfig.RWsubnetName = $RemoteWorkstationSubnetName
+    if( $deployOverDC ) {
+        # Don't create new DC and vnets
+        # prompt for vnet name, gateway subnet name, remote workstatin subnet name, connection service subnet name
+        do {
+            if ( -not $vnetConfig.vnetName ) {
+                $vnetConfig.vnetName = Read-Host "Provide Connection Service Vnet Name"
+            }
+            if ( -not (Find-AzureRmResource -ResourceGroupNameEquals $rgName -ResourceType "Microsoft.Network/virtualNetworks" -ResourceNameEquals $vnetConfig.vnetName) ) {
+                # Does not exist
+                Write-Host ("{0} not found in root resource group {1}" -f ($vnetConfig.vnetName,$rgName))
+                $vnetConfig.vnetName = $null
+            }
+        } while (-not $vnetConfig.vnetName)
+
+        $vnet = Get-AzureRmVirtualNetwork -Name $vnetConfig.vnetName -ResourceGroupName $rgName
+
+        # Connection Service Subnet
+        do {
+            if ( -not $vnetConfig.CSsubnetName ) {
+                $vnetConfig.CSsubnetName = Read-Host "Provide Connection Service Subnet Name"
+            }
+            if ( -not ($vnet.Subnets | ?{$_.Name -eq $vnetConfig.CSsubnetName}) ) {
+                # Does not exist
+                Write-Host ("{0} not found in root resource group VNet {1}" -f ($vnetConfig.CSsubnetName,$vnet.Name))
+                $vnetConfig.CSsubnetName = $null
+            }
+        } while (-not $vnetConfig.CSsubnetName)
+
+        # Gateway Subnet
+        do {
+            if ( -not $vnetConfig.GWsubnetName ) {
+                $vnetConfig.GWsubnetName = Read-Host "Provide Gateway Subnet Name"
+            }
+            if ( -not ($vnet.Subnets | ?{$_.Name -eq $vnetConfig.GWsubnetName}) ) {
+                # Does not exist
+                Write-Host ("{0} not found in root resource group VNet {1}" -f ($vnetConfig.GWsubnetName,$vnet.Name))
+                $vnetConfig.GWsubnetName = $null
+            }
+        } while (-not $vnetConfig.GWsubnetName)        
+        
+        # Remote Workstation Subnet
+        do {
+            if ( -not $vnetConfig.RWsubnetName ) {
+                $vnetConfig.RWsubnetName = Read-Host "Provide Remote Workstation Subnet Name"
+            }
+            if ( -not ($vnet.Subnets | ?{$_.Name -eq $vnetConfig.RWsubnetName}) ) {
+                # Does not exist
+                Write-Host ("{0} not found in root resource group VNet {1}" -f ($vnetConfig.RWsubnetName,$vnet.Name))
+                $vnetConfig.RWsubnetName = $null
+            }
+        } while (-not $vnetConfig.RWsubnetName)
+    } else {
+        #create new DC and vnets
+        if( -not $vnetConfig.vnetName ) {
+            $vnetConfig.vnetName = "vnet-CloudAccessManager"
```

Comment:
I think I'd go with the original names but happy to discuss.

Reply:
i jumbled the default names up when switching between branches, will change back to the correct originals